perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -439,6 +439,32 @@ public ApplicationFormItem call(ApiCaller ac, Deserializer parms) throws PerunEx
 		}
 	},
 
+	/*#
+	 * Get page of applications from the given vo, with the given attributes.
+	 * Query parameter specifies offset, page size, sorting order, sorting column, statuses of the applications,
+	 * whether to include group applications if searching only in VO and string to search
+	 * applications by (by default it searches in group names/descriptions, group and application ids, group uuids,  logins of submitters and
+	 * application_data values), dateFrom and dateTo are optional and specify dates when the applications were submitted,
+	 * the last 2 parameters are optional and serve to search group/member specific applications.
+	 *
+	 * @param vo int Vo <code>id</code>
+	 * @param query ApplicationsPageQuery Query with page information
+	 *
+	 * @return Paginated<Application> page of requested rich applications
+	 * @throw VoNotExistsException if there is no such vo
+	 * @throw PrivilegeException if user doesnt have sufficient privileges
+	 * @throw PerunException
+	 */
+	getApplicationsPage {
+
+		@Override
+		public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
+			return ac.getRegistrarManager().getApplicationsPage(ac.getSession(),
+				ac.getVoById(parms.readInt("vo")),
+				parms.read("query", ApplicationsPageQuery.class));
```

Comment:
I get CANNOT_DESERIALIZE_VALUE error every time I include dateFrom or dateTo in the query, it seems it can't deserialize it correcly if it is part of the object.